diagnostic.md

```diff
@@ -35,7 +42,8 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first will be products/product/:product_id whereas the second will be
+    products/:product_id.
```

Comment:
Can you expand on this?  These two route definitions will be identical URL paths.

One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.